packages/x-license-pro/src/cli/license-cli.ts

```diff
@@ -70,14 +76,17 @@ export function licenseGenCli() {
         const licenseDetails = {
           expiryDate: new Date(new Date().getTime() + parseInt(argv.expiry, 10) * oneDayInMs),
           orderNumber: argv.order,
+          type: argv.type as LicenseScope,
```

Comment:
I'm not following, why mix the type and scope terminology?

Reply:
My bad, I must have missed renaming it here. I've updated it